scripts/cbmc-viewer/tags.py

```diff
@@ -0,0 +1,166 @@
+"""Support linking a symbol to a line of code defining the symbol.
+
+Run etags over the source tree to produce a TAGS file, parse the TAGS
+file, and construct a mapping from a symbol to a line of code defining
+the symbol.
+
+Warning: In source with conditional compilation or otherwise one
+symbol declared in multiple namespaces (eg type vs function) this
+simple trick may fail.  And it does fail a lot.
+"""
+from __future__ import print_function
+
+from builtins import chr
+from builtins import object
+import os
+import re
+import sys
+import errno
+from past.builtins import cmp
```

Comment:
This is imported but not used?

Reply:
Again, this was removed by the conversion to straight Python3 in the second commit of this PR.